gui/WorldView.cxx

```diff
@@ -306,6 +310,17 @@ WorldView::WorldView(QWidget* parent, Qt::WindowFlags flags)
   connect(d->UI.actionShowGroundPlane, SIGNAL(toggled(bool)),
           this, SLOT(setGroundPlaneVisible(bool)));
 
+  for (int i = 0; i < this->children().size(); ++i) {
```

Comment:
On second thought, please don't use this to update the axes; please instead replace the appropriate calls to `d->UI.renderWidget->update();` with `updateGrid()`.

Also, will no longer be applicable, but:

- Don't use C-style casts (in C++), **ever**.
- Always use [`qobject_cast`](http://doc.qt.io/qt-4.8/qobject.html#qobject_cast) to cast a QObject. Use the cast itself to check if the object is of the desired type (result pointer will be null if it isn't).